src/lib.rs

```diff
@@ -99,7 +99,6 @@ where
     );
 
     match mclient.sql(query.as_str(), vec![], r#"{"timeout": 10000}"#, |a| {
```

Comment:
Not really part of this change, but maybe consider moving the SQL statement into its own function, as here:

https://github.com/joyent/boray/blob/master/src/object/update.rs#L108

Or at least extract it from the method. I think it makes it less easier to miss, and update if necessary.

Reply:
Made some changes here.